test/cmdlineTests/standard_eWasm_requested/output.json

```diff
@@ -1,63 +1,4 @@
 {"contracts":{"A":{"C":{"ewasm":{"wast":"(module
-    (import \"ethereum\" \"revert\" (func $eth.revert (param i32 i32)))
```

Comment:
This was remove, because the text format previously just concatenated modules. Can you at least add a comment output by the text format generator into the wast mentioning that ";; Object "xyz" would be encoded as a custom section here"?